zebra/zebra_srte.c

```diff
@@ -133,28 +135,79 @@ static int zebra_sr_policy_notify_update_client(struct zebra_sr_policy *policy,
 	}
 	stream_putl(s, policy->color);
 
+	if (IS_ZEBRA_DEBUG_SRV6) {
+		char endpoint[IPADDR_STRING_SIZE];
```

Comment:
the zebra_sr_policy_notify_update_client() also handles MPLS and displaying traces using srv6 debug guard is inconsistent. 
I would propose to minimise code change by renaming zebra_sr_policy_notify_update_client to zebra_mpls_policy_notify_update_client, and creating a new function zebra_srv6_policy_notify_update_client, which will be called by an upper function named zebra_sr_policy_notify_update_client.